LICENSE

```diff
@@ -1,4 +1,4 @@
-  Copyright 2018 - 2020, Foris Limited ("Crypto.com"). 
+  Copyright 2021, CRO Protocol Labs ("Crypto.org").
```

Comment:
the original code was developed as per that copyright, so perhaps keep both (one 2018-2020, one 2021)?

Reply:
Sure, will do